test/exact_soln/vorticity_qoi.C

```diff
@@ -70,7 +71,9 @@ int main(int argc, char* argv[])
 
   // Initialize libMesh library.
   libMesh::LibMeshInit libmesh_init(argc, argv);
- 
+
+  GRINS::SimulationInitializer initializer;
+
   GRINS::SimulationBuilder sim_builder;
```

Comment:
My intent is for SimulationBuilder to go away after properly factorying out Solver, QoI, etc. (goal is to finish that before tomorrow morning). Also, the PR I'm working on right now will hide all this stuff from the user.